exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -167,24 +167,61 @@ drill.exec: {
     debug: true,
     janino_maxsize: 262144,
     cache_max_size: 1000,
-    // Enable to write generated source to disk. See ClassBuilder
-    save_source: false,
     // Where to save the generated source. See ClassBuilder
     code_dir: "/tmp/drill/codegen"
+    // Disable code cache. Only for testing.
+    disable_cache: false,
+    // Use plain Java compilation where available
+    prefer_plain_java: false
   },
   sort: {
     purge.threshold : 1000,
     external: {
-      batch.size : 4000,
+      // Drill uses the managed External Sort Batch by default.
```

Comment:
The following two changes are not related to DRILL-5116.  Should they be moved to another PR ?